project/LinkerdBuild.scala

```diff
@@ -497,45 +491,42 @@ object LinkerdBuild extends Base {
          |"""
       ).stripMargin
 
-    val MinimalSettings = Defaults.configSettings ++ appPackagingSettings ++ Seq(
+    val BundleSettings = Defaults.configSettings ++ appPackagingSettings ++ Seq(
       mainClass := Some("io.buoyant.linkerd.Main"),
       assemblyExecScript := execScript.split("\n").toSeq,
       dockerEnvPrefix := "L5D_",
-      unmanagedBase := baseDirectory.value / "plugins"
-    )
-
-    val MinimalProjects = Seq[ProjectReference](
-      admin, core, main, configCore, Namer.fs, Protocol.http, Telemetry.tracelog
-    )
-
-    val BundleSettings = MinimalSettings ++ Seq(
+      unmanagedBase := baseDirectory.value / "plugins",
       assemblyJarName in assembly := s"${name.value}-${version.value}-exec",
       dockerTag := version.value
     )
 
     val BundleProjects = Seq[ProjectReference](
-      Namer.consul, Namer.k8s, Namer.marathon, Namer.serversets, Namer.zkLeader, Namer.curator,
+      admin, core, main, configCore,
+      Namer.consul, Namer.fs, Namer.k8s, Namer.marathon, Namer.serversets, Namer.zkLeader, Namer.curator,
       Interpreter.namerd, Interpreter.fs, Interpreter.perHost, Interpreter.k8s,
-      Protocol.h2, Protocol.mux, Protocol.thrift,
+      Protocol.h2, Protocol.http, Protocol.mux, Protocol.thrift,
       Announcer.serversets,
       Telemetry.core, Telemetry.tracelog,
       Tracer.zipkin,
       tls,
       failureAccrual
     )
 
+    val LowmemSettings = BundleSettings ++ Seq(
```

Comment:
Tiniest of nits: can you capitalize this as `LowMemSettings`, to match the `Namerd.LowMemSettings` val?